src/services/icon.service.ts

```
@@ -1,53 +1,54 @@
+				? await StreamDeckIconService.fetchIconWithContentType(iconId)
+				: Promise.reject("Stream Deck icons are disabled");
+		} catch (error) {
+			console.error("Error fetching icon:", error);
+			throw error;
+		}
+	}
+
+	static mkIconUrl(iconId: string): string {
+		return `${iconsEndpoint}/${iconId}`;
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)